worldedit-core/build.gradle.kts

```diff
@@ -84,3 +87,28 @@ sourceSets {
         }
     }
 }
+
+val crowdinApiKey = "crowdin_apikey"
+
+if (project.hasProperty(crowdinApiKey)) {
+    tasks.named<UploadSourceFileTask>("crowdinUpload") {
+        apiKey = "${project.property(crowdinApiKey)}"
+        projectId = "worldedit-core"
+        files = arrayOf(
+                object {
+                    var name = "strings.json"
+                    var source = "$projectDir/src/main/resources/lang/strings.json"
+                }
+        )
+    }
+
+    tasks.named<DownloadTranslationsTask>("crowdinDownload") {
+        apiKey = "${project.property(crowdinApiKey)}"
+        destination = "$projectDir/src/main/resources/lang"
```

Comment:
Similar for resolving files. Also, I feel like this this task should output to the generated code, unless this is intended to be committed to source?